src/files/add-pull-stream.js

```diff
@@ -1,6 +1,13 @@
 'use strict'
 
 const SendFilesStream = require('../utils/send-files-stream')
+const FileStreamConverter = require('../utils/file-stream-converter')
 const toPull = require('stream-to-pull-stream')
 
-module.exports = (send) => (options) => toPull(SendFilesStream(send, 'add')(options))
+module.exports = (send) => {
+ return (options) => {
+   options = options || {}
+   options.converter = FileStreamConverter
```

Comment:
I don't need to, but I could :) In the current way it would make sure that the converter for the PullStream is always the `FileStreamConverter` no matter which value it had. I think that makes sense, doesn't it?